resource/filter.go

```diff
@@ -167,3 +177,24 @@ func (f Filter) matches(r *Resource) bool {
 	}
 	return false
 }
+
+func (f *StringFilter) matches(s string) (matched bool, err error) {
```

Comment:
I guess, we don't need named returned values here, so  `(bool, error)` is ok :+1:

Reply:
Fixed